kymatio/scattering2d/frontend/__init__.py

```diff
@@ -18,9 +18,13 @@ def __init__(self, *args, **kwargs):
 
             # Create frontend-specific class name by inserting frontend name
             # after `Scattering`.
+            frontend = frontend.capitalize()
+            if frontend == 'Numpy':
+                frontend = 'NumPy'
             class_name = self.__class__.__name__
-            class_name = (class_name[:-2] + frontend.capitalize()
-                          + class_name[-2:])
+            class_name = (class_name[:-2] + frontend + class_name[-2:])
+            #print(module, class_name)
+            #raise RuntimeError('\nThe frontend \'' + frontend + '\' could not be correctly imported.')
```

Comment:
I need thoughts on this mess of code.

Reply:
It's in the commit message. Since we're now doing `ScatteringTorch2D` instead of `Scattering2DTorch`, some more acrobatics are necessary.

That being said, however, it won't work here, since the `capitalize()` method doesn't turn `numpy` into `NumPy` (or `tf`/`tensorflow` into `TensorFlow`, for that matter). I think a more robust approach would be to have a dict:
```
frontend_suffixes = {'torch': 'Torch', 'numpy': 'NumPy'}
class_name = 'Scattering' + suffixes[frontend] + '2D'
```

Reply:
Do we want to define this dictionary in some base folder thats not in any of the scattering*d?

Reply:
Might make sense down the road, but we can just leave it here for now.